pkg/trace/api/api.go

```diff
@@ -762,3 +763,34 @@ func getMediaType(req *http.Request) string {
 	}
 	return mt
 }
+
+// getDirector returns a director for a httputil.ReverseProxy. It enriches requests with headers setting custom
+// container and additional tags and overwrites the user agent header.
+func getHeaderDecoratorDirector(tags string, metricSuffix string) func(req *http.Request) {
```

Comment:
In general, in Go, we like to avoid the `get*` prefix (see https://golang.org/doc/effective_go#Getters - granted this isn't a getter). Of course this is not a hard rule and exceptions can be made, but I'd prefer we do it here too.
```suggestion
func headerDecoratorDirector(tags string, metricSuffix string) func(req *http.Request) {
```